server/routerlicious/packages/lambdas/src/scribe/lambda.ts

```diff
@@ -614,6 +614,8 @@ export class ScribeLambda implements IPartitionLambda {
 			sequenceNumber: this.sequenceNumber,
 			validParentSummaries: this.validParentSummaries,
 			isCorrupt: this.isDocumentCorrupt,
+			protocolHead: this.protocolHead,
+			checkpointTimestamp: Date.now(),
```

Comment:
Do we need to add these two new properties and optional in `IScribe`?

Reply:
Looks like it already is in `IScribe`. See `services-core/src/documents.ts`. `protocolHead` is `| undefined`, but we should also make `checkpointTimestamp` `| undefined` for back-compat